changelogs/3.0.0.md

```diff
@@ -0,0 +1,478 @@
+synchronous, set the `is_sync` option to `true`. Setting `is_sync = false` is
+prohibited. If any value other than true/nil is set, for example
+`is_sync = "some string"`, then an error will be thrown (gh-8650).
+
+### Cli
+
+* **[Behavior change]** Disabled starting the Lua REPL by default when running
+  Tarantool. Now, Tarantool yields the message that shows the command usage.
+  To run the Lua REPL, just set the `-i` flag. To pass a Lua script contents via
+  `stdin`, use dash (`-`) as the script name. For more information see a help
+  message by running `tarantool -h` (gh-8613).
+* Introduced new command-line options `--name` and `--config` (gh-8613).
+
+### Compat
+
+* The following compatibility options' defaults were switched to new behavior:
+    * `yaml_pretty_multiline`
+    * `sql_seq_scan_default`
+    * `json_escape_forward_slash`
+    * `fiber_channel_close_mode`
+    * `fiber_slice_default`
+    * `box_cfg_replication_sync_timeout`
+    * `c_func_iproto_multireturn`
+
+  More information on the new behavior can be found on the [Module compat](https://www.tarantool.io/en/doc/latest/reference/reference_lua/compat/) page.
+
+### Console
+
+* Now multiline commands can be used by setting continuation marker
+  using the `\set continuation` command.
+
+### Fiber
+
+* Exported `fiber_set_name_n`, `fiber_name`, `fiber_id`, `fiber_csw` and
+  `fiber_find` into the public C API and usable via FFI as well.
+* Make `fiber_set_joinable`, `fiber_set_ctx` and `fiber_get_ctx`
+  treat the NULL argument as the current fiber.
+
+### Space
+
+* Introduces space type: a new space definition field.
+* Introduces the fully temporary space type. It is the same as data-temporary
+  but also has temporary metadata. Temporary spaces can now be created in
+  read_only mode, they disappear after server restart and don't exist on
+  replicas (gh-8323).
+
+### Test
+
+#### Fuzz
+
+* Added a grammar-based SQL fuzzer (gh-4826).
+
+### Uri
+
+* Introduced the `login` and `password` fields in `uri.parse()` (gh-9435).
+
+### Build
+
+* Tarantool does not depend on libgomp anymore (gh-7689).
+* Added building static `deb` and `rpm` packages.
+* Dropped building distro-specific `deb` and `rpm` Tarantool packages.
+* Dropped building Tarantool packages with GC32. Now only GC64 packages
+  are shipped.
+* Exported all LuaJIT public functions from Tarantool binary (gh-3680).
+
+## Bugs fixed
+
+### Core
+
+* **[Breaking change]** The key `box.space._schema['cluster']` is renamed to
+  `'replicaset_uuid'`. That is not expected to be breaking because `_schema` is
+  an internal system space, but the key was visible in public and documented
+  (gh-5029).
+* **[Breaking change]** The table `box.info.cluster` is renamed to
+  `box.info.replicaset`. The behaviour can be reverted using the `compat` option
+  `box_info_cluster_meaning`
+  (https://tarantool.io/compat/box_info_cluster_meaning) (gh-5029).
```

Comment:
The page is not created yet: https://github.com/tarantool/doc/issues/3465.

Reply:
We just started rolling docs. It will appears soon. @andreyaksenov could you double check the address will be valid?

Reply:
The `https://tarantool.io/compat/box_info_cluster_meaning` page doesn't exist in docs. It is better to add a link to the root `Module compat` topic: https://www.tarantool.io/en/doc/latest/reference/reference_lua/compat/

Reply:
We can update this as follows:

```
The behaviour can be reverted using the `compat` option
  `box_info_cluster_meaning`
  (https://tarantool.io/compat/box_info_cluster_meaning) (gh-5029).
  
->

The behaviour can be reverted using the [compat](https://www.tarantool.io/en/doc/latest/reference/reference_lua/compat/) option
  `box_info_cluster_meaning` (gh-5029).
```

Reply:
agreed to remove direct link.